linera-base/src/data_types.rs

```diff
@@ -652,16 +652,31 @@ impl Amount {
 }
 
 /// Permissions for applications on a chain.
-#[derive(Default, Debug, PartialEq, Eq, Hash, Clone, Serialize, Deserialize)]
+#[derive(
+    Default,
+    Debug,
+    PartialEq,
+    Eq,
+    Hash,
+    Clone,
+    Serialize,
+    Deserialize,
+    WitType,
+    WitLoad,
+    WitStore,
+    InputObject,
+)]
 pub struct ApplicationPermissions {
     /// If this is `None`, all system operations and application operations are allowed.
     /// If it is `Some`, only operations from the specified applications are allowed, and
     /// no system operations.
     pub execute_operations: Option<Vec<ApplicationId>>,
```

Comment:
Kind of unrelated to this PR, but wouldn't something like `operations_allowed_from` or something like that? At least for me, when I read `execute_operations`, it makes me think that I need to pass some operations to be executed